src/main/java/InsertionSort/InsertionSort.java

```diff
@@ -0,0 +1,19 @@
+package InsertionSort;
+
+public class InsertionSort {
+
+//got inspiration for this code from (https://www.geeksforgeeks.org/insertion-sort/)
+    public static int[] InsertionSort(int[] list) {

```

Comment:
I rarely comment on variable names, but this variable name is misleading - an array called list? it implies the int array is a list - this is confusing. I recommend revisiting this choice.